pages/items.js

```diff
@@ -20,7 +21,17 @@ export default function ItemsPage() {
     return <h1>error! fehlerhafte Daten.</h1>;
   }
 
-  const sortedItems = items.slice().sort((a, b) => {
+  const filteredItems = items.filter((item) => {
+    if (statusFilter === "!isSold") {
+      return !item.isSold;
+    } else if (statusFilter === "isSold") {
+      return item.isSold;
+    } else if (statusFilter === "all") {
+      return item;
```

Comment:
i personally would have done something like isSold === true, and add a state for sold, but it also works and makes sense the way you did it 💪

Reply:
if you meant line 28, i hat it like this before, but i think it does the same just a tiny bit shorter ✌🏼